components/files/file_features/file_features.py

```diff
@@ -47,21 +54,37 @@ def query_file_features(a_server_config, a_site_id, a_file_upload_uuid, a_file_n
     logging.debug ("get feature job returned {0}\n{1}".format(response.status_code, json.dumps(rj,indent=4)))
 
     # Select all returned features to post an import job. This will extract the design data from the file.
-    file_uuid = rj["file_uuid"]
     features_to_import = []
     feature_list = rj["features"]
     
-
     for i, feature in enumerate(feature_list):
     
         v = {k : feature.get(k) for k in ["design_type", "filter", "source_name"] if k in feature}
         features_to_import.append(v)
     return features_to_import
 
 
+
+
+# This function runs a process in the design file microservice called an "import job".
+# Import jobs perform the following steps:
+# 1. Import a list of design objects (which can be determined by the feature job in query_file_features) into Sitelink3D v2. 
+# 2. Add RDM representations for the imported design objects so that they can be found and referenced within Sitelink3D v2.

```

Comment:
The designfile service doesn't create the RDM objects, the client does. I can see the RDM payload being produced (line 132) but it doesn't look like anything is been done with it.

Reply:
Sorry not sure what you mean by nothing being done with it. This payload is encoded at line 147 and posted at 150.